pkg/gatherers/clusterconfig/gather_config_maps.go

```diff
@@ -42,9 +43,15 @@ import (
 // - 4.4.6+
 //
 // ### Changes
-// - `cluster-monitoring-config` data since versions 4.6.22+ and 4.7.0+
-// - `cluster-config-v1` since versions 4.9.0+
-// - `gateway-mode-config` data from 'openshift-network-operator' namespace since 4.14.0+
+// - `cluster-monitoring-config` config map from `openshift-monitoring`
+//    namespace since versions 4.6.22+ and 4.7.0+
+// - `install-config` key from the `cluster-config-v1` config map in the
+//   `kube-system` namespace since versions 4.9.0+; originally stored as
+//   `config/configmaps/kube-system/cluster-config-v1.json`, changed to
+//   `config/configmaps/kube-system/cluster-config-v1/install-config` since
+//   4.13.0+
+// - `gateway-mode-config` config map from `openshift-network-operator`
+//   namespace since 4.14.0+
```

Comment:
Something like following should hopefully fix the lint problem

```suggestion
//   - `cluster-monitoring-config` config map from `openshift-monitoring`
//      namespace since versions 4.6.22+ and 4.7.0+
//   - `install-config` key from the `cluster-config-v1` config map in the
//     `kube-system` namespace since versions 4.9.0+; originally stored as
//     `config/configmaps/kube-system/cluster-config-v1.json`, changed to
//     `config/configmaps/kube-system/cluster-config-v1/install-config` since
//     4.13.0+
//   - `gateway-mode-config` config map from `openshift-network-operator`
//     namespace since 4.14.0+

```

Reply:
Thanks for the advice. Fixed. Sorry for cutting the corner there. :innocent: